tests/test_uncertain_conditional.py

```diff
@@ -261,16 +240,15 @@ def mean_sq_fn(X):
     mean_sq_quad = mvnquad(mean_sq_fn, *quad_args)
     var_quad = var_quad + (mean_sq_quad - mean_quad**2)
 
-    mean_analytic, var_analytic = uncertain_conditional(
-        DataQuad.Xmu,
-        DataQuad.Xvar,
-        inducing_variable,
-        kernel,
-        DataQuad.q_mu,
-        DataQuad.q_sqrt,
-        mean_function=mean_function,
-        full_output_cov=False,
-        white=white)
+    mean_analytic, var_analytic = uncertain_conditional(DataQuad.Xmu,
+                                                        DataQuad.Xvar,
+                                                        inducing_variable,
+                                                        kernel,
+                                                        DataQuad.q_mu,
+                                                        DataQuad.q_sqrt,
+                                                        mean_function=mean_function,
+                                                        full_output_cov=False,
+                                                        white=white)
```

Comment:
Do different people have different settings for their editors(' auto-rejustify)? It seems like unrelated code keeps changing indentation and whether arguments are stuffed on one or two lines vs one argument per line - I find this quite distracting from reviewing PRs, would it be possible to avoid having this happen? Like a dedicated PR that *only* changes the whitespace, and gets everything into one consistent format (that we would have to agree on beforehand!)?

Reply:
`.style.yapf`


```
[style]
based_on_style: pep8
indent_width: 4
split_before_logical_operator = true
column_limit = 100
```

Reply:
We still shouldn't have drive-by formatting changes, this makes PRs significantly more unclear.

Reply:
Yes, agree. Sorry for that, I try to keep changes minimal, but sometimes fail :)